src/ploomber/telemetry/telemetry.py

```diff
@@ -62,6 +63,50 @@ def is_online(host='http://ploomber.io'):
         return False
 
 
+# Will output if the code is within a container
+def is_docker():
+    path = '/proc/self/cgroup'
+    return (os.path.exists('/.dockerenv')
+            or os.path.isfile(path) and any('docker' in line
+                                            for line in open(path)))
+
+
+# The function will output the client platform
+def get_os():
+    os = platform.system()
+    if os == "Darwin":
+        return 'MacOS'
+    else:  # Windows/Linux are contained
+        return os
+
+
+# The function will tell if the code is running in a conda env
+def is_conda():
+    return os.path.exists(os.path.join(sys.prefix, 'conda-meta')) or \
+            bool(os.environ.get("CONDA_PREFIX")) or \
```

Comment:
no need to use `bool()` here `None` evaluates to `False`. Also, the other modules use parenthesis to break down large statements (instead of `\`)